### ISSUE TITLE
Forgotten `format()` in IntegerSerializer

### ISSUE THREAD

**Issue by danie1k** [EliotBerriot/django-dynamic-preferences#133]
In
```python
class IntegerSerializer(BaseSerializer):

    @classmethod
    def clean_to_db_value(cls, value):
        if not isinstance(value, int):
            raise cls.exception('IntSerializer can only serialize int values')
        return value

    @classmethod
    def to_python(cls, value, **kwargs):
        try:
            return int(value)
        except:
            raise cls.exception("Value {0} cannot be converted to int")
```
You've forgotten `.format(value)` for 
```python
raise cls.exception("Value {0} cannot be converted to int")
```

PS.
And you have mixed double quotes with single quotes.

**Comment by EliotBerriot** [comment]
Hi and thank you for the bug report, do you want to open a PR for this? Otherwise I can do it :)

**Comment by danie1k** [comment]
Ok, I can do that, give me few more days :)

**Comment by EliotBerriot** [comment]
Sure, there is no rush!

**Comment by EliotBerriot** [comment]
Fixed in #138
